prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
```

Comment:
If the unassign fails then something is wrong with prow. We should log an error and not post a comment.